compiler/luci-interpreter/src/kernels/Tanh.test.cpp

```diff
@@ -97,6 +97,52 @@ TEST(TanhTest, Uint8)
   EXPECT_THAT(extractTensorShape(output_tensor), ::testing::ElementsAreArray(ref_output_shape));
 }
 
+TEST(TanhTest, InputTypeInvalid_NEG)
+{
+  std::vector<int64_t> input_data{
+      0,  -6, 2, 4, //
+      -4, -2, 8, 1, //
+      0,  -6, 2, 4, //
+      -4, -2, 8, 1, //
+      0,  -6, 2, 4, //
+      -4, -2, 8, 1, //
+      0,  -6, 2, 4, //
+      -4, -2, 8, 1, //
+      0,  -6, 2, 4, //
+      -4, -2, 8, 1, //
+      0,  -6, 2, 4, //
+      -4, -2, 8, 1, //
+  };
+  Tensor input_tensor = makeInputTensor<DataType::S64>({2, 6, 4, 1}, input_data);
+  Tensor output_tensor = makeOutputTensor(DataType::FLOAT32);
```

Comment:
Not sure, but maybe you mentioned this? (suggestion below is part of this)
```suggestion
  Tensor output_tensor = makeOutputTensor(DataType::S64);
```

Reply:
Thank you for your advice, @binarman :+1: 
I will reflect this idea from later tasks! :)